src/main/resources/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookTriggerImpl/config.groovy

```diff
@@ -0,0 +1,15 @@
+package com.atlassian.bitbucket.jenkins.internal.trigger.BitbucketWebhookTriggerImpl
+def f = namespace(lib.FormTagLib)
+
+f.section() {
+
+    f.entry(title: _("bitbucket.trigger.warning"), name: "warning", inline: "true") {}
+
+    f.entry(title: _("bitbucket.trigger.refTrigger"), field: "refTrigger", name: "refTrigger", inline: "true") {
+        f.checkbox(checked:instance == null ? "true": instance.refTrigger)
```

Comment:
this ensures it is ticked by default. When this code is run for a _new_ job we don't have an instance to query state from, so this works around that.

Reply:
Is there a reason we can't do this?
`f.checkbox(default: "true")`
The default property looks like it does an instance null check: https://reports.jenkins.io/core-taglib/jelly-taglib-ref.html#form:checkbox

Reply:
I'll give it a try! I've tried so many things I can no longer remember what worked and what didn't, `default:true` seems promising.